정민지/mvvmstudy/app/build.gradle

```
@@ -0,0 +1,55 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        applicationId "com.github.sookhee.mvvmstudy"
+        minSdk 21
+        targetSdk 31
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    viewBinding {
+        enabled = true
+    }
+
```

이게 안드로이드 한글 공식 문서에는 최신 변경사항이 안나오더라구요 ㅠ
```suggestion
   buildFeatures {
        viewBinding true
   }
```

기능 동작으로는 차이점이 없어서 참고만 해주세요!

---

오오 그렇군요, 감사합니다!